managed/ui/src/components/tables/ListBackups/ListBackups.js

```diff
@@ -283,6 +285,42 @@ export default class ListBackups extends Component {
     });
   };
 
+  /**
+   * This method will help us to get the particular backupUUID
+   * for the selected row.
+   *
+   * @param {string} {backupUUID} Backup UUID.
+   * @param {boolean} isSelected Selected row.
+   * @returns Boolean
+   */
+  onRowSelect = ({ backupUUID }, isSelected) => {
+    isSelected ?
+      this.setState({ selected: [...this.state.selected, backupUUID].sort() }) :
+      this.setState({ selected: this.state.selected.filter((id) => id !== backupUUID) });
+
+    return false;
```

Comment:
Return value should most likely be true even if you are using state to select the row.

Reply:
Done